### ISSUE TITLE
Bad parsing of infoboxes with tab separators

### ISSUE THREAD

**Issue by tmtmtmtm** [spencermountain/wtf_wikipedia#435]
When parsing https://en.wikipedia.org/w/index.php?title=Lino_Spiteri&oldid=1027940292, the infobox gets mangled into:

```json
      "infoboxes": [
        {
          "[object Object]": {
            "text": "office1\t\t= Finance Minister of Malta",
            "links": [
              {
                "type": "internal",
                "page": "Finance Minister of Malta"
              }
            ]
          },
          "alma_mater": {
            "text": "St Peter's College, Oxford",
            "links": [
              {
                "type": "internal",
                "page": "St Peter's College, Oxford"
              }
            ]
          }
        }
      ],
```

This seems to be because of the use of tabs as separators, as after replacing them all with spaces in the source, everything works fine again.

**Comment by spencermountain** [comment]
ah interesting bug, thank you Tony.
busy right now, but will add this to the queue. prs welecome

**Comment by spencermountain** [comment]
fixed in 10.0.4
cheers
